### PR TITLE
Steves dev branch/tasks

### PR DESCRIPTION
**Pull request by boothscript**

### CODE REVIEW

**Review by jennifer-carey — DISMISSED** [review]
Great - approved

**Review by netty-carroll — APPROVED** [review]
looks good!

**Review by nisharai26 — APPROVED** [review]
Good

**Review by nisharai26 — DISMISSED** [review]
Great!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-# master21-git101
+
```
Comment (tamjankowska) [inline README.md:1]: Can you include your name here plz and remove any white space.

**Review by nmunro — APPROVED** [review]

**Review by tamjankowska — APPROVED** [review]
Watch out for your whitespace ;)

**Review by JakeyD11 — APPROVED** [review]

**Pull request merged**